package.json

```
@@ -4,12 +4,12 @@
   "description": "A module for Fastify to parse x-www-form-urlencoded bodies",
   "main": "formbody.js",
   "scripts": {
-    "test": "tap test/**/*.test.js",
+    "test": "tap test/**/*.test.js && tsd",
```

there was `test-ci` called, we can include it in the `test` script

```suggestion
    "test": "standard && tap test/**/*.test.js && tsd",
```